static/js/tools/scatter/chart.tsx

```diff
@@ -141,6 +141,22 @@ export function Chart(props: ChartPropsType): JSX.Element {
     };
   }, [props, chartContainerRef, geoJsonFetched]);
 
+  // Triggered only when stat vars or places change to avoid double counting and send data to google analytics.
+  let statVar1 = "";
+  let statVar2 = "";
+  if (props.facetList.length >= 2) {
+    statVar1 = props.facetList[0].dcid;
+    statVar2 = props.facetList[1].dcid;
+  }
+  useEffect(() => {
+    window &&
+      window.gtag &&
+      window.gtag("event", "tool_chart_plot", {
+        place_dcid: props.placeInfo.enclosingPlace.dcid,
+        stat_var: [statVar1, statVar2],
+      });
+  }, [statVar1, statVar2, props.placeInfo.enclosingPlace.dcid]);
```

Comment:
I thought about that too. The problem with the props.facetList is it includes metadata. If the source of a stat var changes, it will be triggered as well. However, for this event, we may want to focus more on what combination of stat vars and place dcids are plotted.
For props.xLabel and props.yLabel, they are different from stat var dcid format. Or do you mean we can look at changes to props.xlabel and props.ylabel and return statvar1 and statvar2 value?

Reply:
ah, I see what you mean, ok, keeping it the way it is makes sense!